src/screens/internal/ShareImage/ChartShareImage.tsx

```diff
@@ -9,7 +9,7 @@ import {
   Subtitle,
   Title,
 } from './ChartShareImage.style';
-import { ScreenshotWrapper } from './ShareImage.style';
+import { DarkScreenshotWrapper } from './ShareImage.style';
```

Comment:
I refactored `ScreenshotWrapper` into `DarkScreenshotWrapper` and `ScreenshotWrapper`. All of the existing usages are `DarkScreenshotWrapper` now.